Source/ThirdParty/gtest/xcode/gtest.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 46;
+	objectVersion = 52;
```

Comment:
I never know if this is OK, or breaks older Xcodes.

Reply:
This version number is precedented in some other WebKit projects, so I assume it is not breaking any version of Xcode we care about.